manifests/params.pp

```diff
@@ -7,6 +7,10 @@
   $ensure_service                = 'running'
   $enable_service                = true
   $cache_mem                     = '256 MB'
+  $visible_hostname              = 'mypc@workgroup'
+  $via                           = true
```

Comment:
This *is* the default in squid, but perhaps it'd be better to default to `undef` and not write anything into the configuration file by default.

That way, nothing will break for users who are already setting these options by using `extra_config_sections`.